quarkus/admin/distribution/NOTICE

```diff
@@ -1163,36 +1163,36 @@ NOTICE:
 
 --------------------------------------------------------------------------------
 
-Group: software.amazon.awssdk Name: apache-client Version: 2.31.30
-Group: software.amazon.awssdk Name: arns Version: 2.31.30
-Group: software.amazon.awssdk Name: auth Version: 2.31.30
-Group: software.amazon.awssdk Name: aws-core Version: 2.31.30
-Group: software.amazon.awssdk Name: aws-query-protocol Version: 2.31.30
-Group: software.amazon.awssdk Name: aws-xml-protocol Version: 2.31.30
-Group: software.amazon.awssdk Name: checksums Version: 2.31.30
-Group: software.amazon.awssdk Name: checksums-spi Version: 2.31.30
-Group: software.amazon.awssdk Name: crt-core Version: 2.31.30
-Group: software.amazon.awssdk Name: endpoints-spi Version: 2.31.30
-Group: software.amazon.awssdk Name: http-auth Version: 2.31.30
-Group: software.amazon.awssdk Name: http-auth-aws Version: 2.31.30
-Group: software.amazon.awssdk Name: http-auth-aws-eventstream Version: 2.31.30
-Group: software.amazon.awssdk Name: http-auth-spi Version: 2.31.30
-Group: software.amazon.awssdk Name: http-client-spi Version: 2.31.30
-Group: software.amazon.awssdk Name: iam-policy-builder Version: 2.31.30
-Group: software.amazon.awssdk Name: identity-spi Version: 2.31.30
-Group: software.amazon.awssdk Name: json-utils Version: 2.31.30
-Group: software.amazon.awssdk Name: metrics-spi Version: 2.31.30
-Group: software.amazon.awssdk Name: netty-nio-client Version: 2.31.30
-Group: software.amazon.awssdk Name: profiles Version: 2.31.30
-Group: software.amazon.awssdk Name: protocol-core Version: 2.31.30
-Group: software.amazon.awssdk Name: regions Version: 2.31.30
-Group: software.amazon.awssdk Name: retries Version: 2.31.30
-Group: software.amazon.awssdk Name: retries-spi Version: 2.31.30
-Group: software.amazon.awssdk Name: s3 Version: 2.31.30
-Group: software.amazon.awssdk Name: sdk-core Version: 2.31.30
-Group: software.amazon.awssdk Name: sts Version: 2.31.30
-Group: software.amazon.awssdk Name: third-party-jackson-core Version: 2.31.30
-Group: software.amazon.awssdk Name: utils Version: 2.31.30
+Group: software.amazon.awssdk Name: apache-client Version: 2.31.40
```

Comment:
The version here changed, does that mean we missed an AWS bump in the notice?

Reply:
Yes, I ran my script again and I found that several versions in `LICENSE` and `NOTICE` were not up to date.